soc/xtensa/intel_s1000/memory.h

```diff
@@ -6,30 +6,40 @@
 #ifndef __INC_MEMORY_H
 #define __INC_MEMORY_H
 
+#include <autoconf.h>
```

Comment:
Why do you need autoconf.h here?

Reply:
The autoconf.h is required for dereferencing CONFIG_BOOTLOADER_MCUBOOT.